admin/src/components/Rectifications/Rectifications.js

```diff
@@ -5,10 +5,36 @@ import { Tab, Tabs, TabList, TabPanel } from 'react-tabs';
 import 'react-tabs/style/react-tabs.css';
 import 'theme/Tabs.css';
 
+import Table from 'components/core/Common/Table/Table';
 import Loader from 'components/core/cards/dashboard/Loader';
 import Card from 'components/core/cards/dashboard/Card';
 import Paginate from 'components/core/Paginate';
 
+const actions = [
+  {
+    label: 'Details',
+    onClick: (e, item) => {
```

Comment:
you'll have to resolve conflicts here :(

Reply:
Yep, makes sense :)

I will rebase and let you know of the changes